script/pre-commit

```diff
@@ -38,7 +38,7 @@ if [[ ! -x "${CLANG_FORMAT}" ]]; then
 fi
 
 CLANG_FORMAT_VERSION="$(${CLANG_FORMAT} -version | cut -d ' ' -f 3)"
-CLANG_FORMAT_VERSION_REQUIRED="3.8.0"
+CLANG_FORMAT_VERSION_REQUIRED="5.0.0"
```

Comment:
I thought this version change is already in.    If no, can we do it in a separate PR.

Reply:
I think you're on old diff...